docs/fields-configuration.md

```diff
@@ -12,7 +12,7 @@ For each config entry the following fields are available:
 - `name` *mandatory*: dotted path field, matching an entry in [Fields definition](./glossary.md#fields-definition)
 - `fuzziness` *optional (`long` and `double` type only)*: when generating data you could want generated values to change in a known interval. Fuzziness allow to specify the maximum delta a generated value can have from the previous value (for the same field), as a delta percentage; value must be between 0.0 and 1.0, where 0 is 0% and 1 is 100%. When not specified there is no constraint on the generated values, boundaries will be defined by the underlying field type
 - `range` *optional (`long` and `double` type only)*: value will be generated between `min` and `max`
-- `cardinality` *optional*: distribution of different values for the field, expressed as a ratio between a `numerator` and a `denominator`
+- `cardinality` *optional*: number of different values for the field
```

Comment:
What is going to happen if cardinality is greater than the number of events being created? My understanding is that this PR does not fix this behaviour, so if I specify i.e. `cardinality: 1000` but generate `100` events I will get `cardinality: 100` (which is fair, no complaints here). If my understanding is correct I would just mention this here
```suggestion
- `cardinality` *optional*: number of different values for the field; note that this value may not be respected if not enough events are generated. Es `cardinality: 1000` with `100` generated events would produce `100` different values, not `1000`.
```